eth_verifier/test/Fields.t.sol

```diff
@@ -8,40 +8,35 @@ import "../lib/bn254/BN256G2.sol";
 import "../lib/bn254/BN254.sol";
 import "../lib/Oracles.sol";
 
-using {Base.add, Base.mul} for Base.FE;
-using {Scalar.add, Scalar.mul, Scalar.inv} for Scalar.FE;
 using {Oracles.to_field} for Oracles.ScalarChallenge;
 
 contract FieldsTest is Test {
     function test_add_base() public {
-        Base.FE q = Base.FE.wrap(Base.MODULUS - 1);
-        Base.FE one = Base.FE.wrap(1);
+        uint256 q = Base.MODULUS - 1;
+        uint256 one = 1;
```

Comment:
We can remove this variable and replace it where used.

Reply:
done